.github/workflows/publish-bufs.yaml

```diff
@@ -24,7 +24,8 @@ jobs:
       # Only deletions are allowed. Breaking updates must happen in a new version of the package(s) in question.
       # This is just a sanity check!
       # For example it allows you to delete files from the latest version of a package (depending on context that might even be ok).
-      run: buf breaking --against '.git#ref=HEAD^' --error-format json | jq .type | if grep -v -q \"FILE_NO_DELETE\"; then exit 1; fi
+      # TODO (arm-versioning): Arm versioning by remove the comment of the next line
```

Comment:
I think you have to comment out the name line as well

Reply:
Yes. sorry.